code/handlers/trackservers.q

```diff
@@ -125,7 +121,7 @@ addnthawc:{[name;proctype;hpup;attributes;W;checkhandle]
     if[checkhandle and not isalive:.dotz.liveh W;'"invalid handle"];
     cleanup[];
     $[not hpup in (exec hpup from .servers.SERVERS) inter (exec hpup from .servers.nontorqprocesstab);
-        `.servers.SERVERS insert(name;proctype;lower hpup;W;0i;$[isalive;.proc.cp[];0Np];.proc.cp[];0Np;attributes);
+        [`.servers.SERVERS insert(name;proctype;lower hpup;W;0i;$[isalive;.proc.cp[];0Np];.proc.cp[];0Np;attributes);.servers.SERVERS];
```

Comment:
Would ``value`.servers.SERVERS insert...`` avoid the need for this code block and returning the servers separately?

Reply:
Got it. Why do you only want to return `.servers.SERVERS` if this is true? Looks like value doesn't get used as this function outputs `W`.

Reply:
I don't think there's a particular reason for that.  It was in torq from before. It doesn't seem to be used anywhere else. Probably it was used in the past and nobody changed it when they decided to output just the handle from the function. I think in can be removed from the code.

Reply:
It's not in master, the change was made in this branch.